src/components/errors/ErrorNetworkMismatch.tsx

```diff
@@ -4,12 +4,21 @@ import { getChainName } from '@dcl/schemas/dist/dapps/chain-id'
 import { ErrorContainer, ErrorDetails, ErrorImage } from './Error'
 import errorImage from '../../images/errors/robotsmiling.png'
 import { switchToChainId } from '../../eth/provider'
+import { track } from '../../utils/tracking'
 import './errors.css'
 
 export interface ErrorNetworkMismatchProps {
   extra: Record<string, any>
 }
 
+const switchChain = (options: Record<string, any>) => {
```

Comment:
Isn't the typing of options a bit loose? We could call this function like this:

```ts
switchChain({ something: 22, breaking: { stuff: 2 } })
```

Typescript'd say everything's fine, but the call would blow up. Isn't it better to just receive `wantedChainId` and `providerChainId`?